src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1204,8 +1205,18 @@ private void onDecodeConnect(
                 reasonCode = PROTOCOL_ERROR;
             }
 
+            final String16FW clientIdentifier = packet.clientId();
+
+            mqttPropertyRW.wrap(propertyBuffer, 0, propertyBuffer.capacity());
```

Comment:
Use return value `mqttProperty` on stack rather than repeated references to field.